scripts/update_identifiers_for_existing_things.py

```diff
@@ -0,0 +1,102 @@
+from time import sleep
```

Comment:
This was the script that backfilled the data for our existing Things db.  I ran it on mars today and all was well.